fixtures/workflow/src/index.ts

```diff
@@ -31,8 +32,35 @@ export class Demo extends WorkflowEntrypoint<{}, Params> {
 	}
 }
 
+export class Demo2 extends WorkflowEntrypoint<{}, Params> {
+	async run(event: WorkflowEvent<Params>, step: WorkflowStep) {
+		const { timestamp, payload } = event;
+
+		const result = await step.do("First step", async function () {
+			return {
+				output: "First step result",
+			};
+		});
+
+		// @ts-expect-error need workers-types aaaaa
```

Comment:
I imagine this PR won't be merged until workerd is released, so this will need removed at some point